### PR TITLE
Add support for configuring podman

### PR DESCRIPTION
**Pull request by jouvin**
Based on Docker feature. Tested successfully at IJCLab.

### CODE REVIEW

File: features/podman/api_daemon.pan [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+'/software/components/systemd/skip/service' = false;
+
+# This is a customization file
+'/software/components/systemd/unit/podman/file/replace' = false;
+
+prefix '/software/components/systemd/unit/podman/file/config/service';
+'EnvironmentFile' = list('/etc/sysconfig/podman');
+
+'/software/components/systemd/unit' = {
+    SELF['podman'] = nlist('startstop', true);
```
Comment (jrha) [inline features/podman/api_daemon.pan:14]:
```suggestion
    SELF['podman'] = dict('startstop', true);
```

File: features/podman/api_daemon.pan [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    SELF['podman']['state'] = 'enabled';
+    SELF;
+};
```
Comment (jrha) [inline features/podman/api_daemon.pan:17]: Or alternatively:
```suggestion
prefix '/software/components/systemd/unit/podman';
'startstop' = true;
'state' = 'enabled';
```

Comment (jouvin) [reply features/podman/api_daemon.pan]: The problem is that the service name is not the same on EL7 and EL8 so we cannot use the `prefix` construct...

File: features/podman/api_daemon.pan [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+include 'components/systemd/config';
+
+'/software/components/systemd/skip/service' = false;
+
+'/software/components/systemd/unit' = {
+    SELF[PODMAN_SERVICE]['file']['replace'] = false;
+    SELF[PODMAN_SERVICE]['file']['config']['service'] = dict('EnvironmentFile', list('/etc/sysconfig/podman'));
+    SELF[PODMAN_SERVICE] = dict('startstop', true);
```
Comment (jrha) [inline features/podman/api_daemon.pan:16]:
```suggestion
    SELF[PODMAN_SERVICE]['startstop'] = true;
```
Othewise this overwrites the previous two lines?

**Review by jrha — APPROVED** [review]
LGTM

**Pull request merged**